bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -21,23 +36,212 @@ log:set DEBUG org.openhab.core.automation
 
 For more information on the available APIs in scripts see the [JSR223 Scripting]({{base}}/configuration/jsr223.html) documentation.
 
-## Script Examples
+The following examples show how to access common openHAB functionalities.
+
+## Logging
 
-JavaScript scripts provide access to almost all the functionality in an openHAB runtime environment.
 As a simple example, the following script logs "Hello, World!".
 Note that `console.log` will usually not work since the output has no terminal to display the text.
 The openHAB server uses the [SLF4J](https://www.slf4j.org/) library for logging.
 
-```js
-const LoggerFactory = Java.type('org.slf4j.LoggerFactory');
-
-LoggerFactory.getLogger("org.openhab.core.automation.examples").info("Hello world!");
+```javascript
+let logger = Java.type('org.slf4j.LoggerFactory').getLogger('org.openhab.rule.<ruleId>');
+logger.info('Hello world!');
+logger.warn('Successfully logged warning.');
+logger.error('Successfully logged error.');
 ```
-
-Depending on the openHAB logging configuration, you may need to prefix logger names with `org.openhab.core.automation` for them to show up in the log file (or you modify the logging configuration).
+Replace `<ruleId>` with your rule's or script's (unique-)id.
 
 The script uses the [LoggerFactory](https://www.slf4j.org/apidocs/org/slf4j/Logger.html) to obtain a named logger and then logs a message like:
 
 ```text
-    ... [INFO ] [.openhab.core.automation.examples:-2   ] - Hello world!
+    ... [INFO ] [org.openhab.rule.<ruleId>   ] - Hello world!
+    ... [WARN ] [org.openhab.rule.<ruleId>   ] - Successfully logged warning.
+    ... [ERROR] [org.openhab.rule.<ruleId>   ] - Successfully logged error.
+```
+
+## Core Actions
+
+The basic openHAB services, which are pre-included in JavaScript actions when this Add-On is __not__ installed, allow the access to items and more.
+
+```javascript
+let { itemRegistry, things, rules, events, actions } = require('@runtime');
```

Comment:
Instead of those single „imports“, use the default scope (from the [openhab-core/DefaultScriptScopeProvider.java](https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/internal/defaultscope/DefaultScriptScopeProvider.java) with 
```javascript
let openhab = require('@runtime');
```

Reference `itemRegistry` then with `openhab.itemRegistry` and so on…